docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -0,0 +1,236 @@
+# Product Editor Extensibility Guidelines
+
+> ⚠️ **Notice:** These guidelines are currently a **work in progress**. Please be aware that some details might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
+
+Changelog:
+
+- 2021-11-17: Initial version
+
+## Form Extensibility
+
+**Our research shows that merchants think in terms of tasks and goals while creating new products in Woo.** For example, adding or editing a product price is a separate task from uploading images or managing inventory.
+
+For this reason, the new product form features groups (tabs) that correspond with these tasks. Simply speaking, they are separate views where different form features live. As a developer, you can extend any of these views and offer merchants the information and tools they need to create better, more successful products.
+
+There are several ways to extend the new product form: from a single field to a whole group or section containing multiple fields and tables. These extension points are tightly linked to the form structure, giving you plenty of freedom to create a great user experience.
```

Comment:
I just wondered if the word 'tightly' in 'tightly linked' is serving any purpose here. It almost sounds like a bad thing. Maybe just 'linked' would be enough? Your call :)

Reply:
I agree, I'll remove "tightly". Good call.